test/unit/Main.hs

```diff
@@ -26,6 +26,7 @@ import TestLanguagePipeline (testLanguagePipeline)
 import TestModel (testModel)
 import TestNotification (testNotification)
 import TestPretty (testPrettyConst)
+import TestVoid (testVoid)
```

Comment:
It's probably overkill to have a separate test module just for testing `void`.  I would put the `void` tests in `TestLanguagePipeline`, perhaps?